src/main/webapp/test/mashup.jsp

```diff
@@ -8,429 +8,520 @@
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Admin Home Page</div>
+        <div id="adminHomePage"></div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Admin Search Page</div>
+        <div id="adminSearchPage"></div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Admin Activity Log Page</div>
+        <div id="adminActivityLogPage"></div>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Home Page</div>
+        <div id="index">
+          <iframe class="full-width" src="/index.jsp"></iframe>
+        </div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Features Page</div>
+        <div id="features">
+          <iframe class="full-width" src="/features.jsp"></iframe>
+        </div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">About Us Page</div>
+        <div id="about">
+          <iframe class="full-width" src="/about.jsp"></iframe>
+        </div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Contact Page</div>
+        <div id="contact">
+          <iframe class="full-width" src="/contact.jsp"></iframe>
+        </div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Terms Of Use Page</div>
+        <div id="terms">
+          <iframe class="full-width" src="/terms.jsp"></iframe>
+        </div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Request Account Page</div>
+        <div id="request">
+          <iframe class="full-width" src="/request.jsp"></iframe>
+        </div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Usermap Page</div>
+        <div id="usermap">
+          <iframe class="full-width" src="/usermap.jsp"></iframe>
+        </div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Student Help Page</div>
+        <div id="studentHelp">
+          <iframe class="full-width" src="/studentHelp.jsp"></iframe>
+        </div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Instructor Help Page</div>
+        <div id="instructorHelp">
+          <iframe class="full-width" src="/instructorHelp.jsp"></iframe>
+        </div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Deadline Exceeded Error Page</div>
+        <div id="deadlineExceededErrorPage"></div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Error Page</div>
+        <div id="errorPage"></div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Entity Not Found Page</div>
+        <div id="entityNotFoundPage"></div>
+        <br><hr class="hr-bold"><br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+      </div>
```

Comment:
Extraneous `</div>`. There are actually 93, not 94, opening `<div>`s - there's a false positive in the `<script>` section.